extended the intro

pletzer opened the pull request:
Hi @chrisdjscott and @mand35, 

+ Added section on how to run the code
+ Added subsection on how to adjust the problem size
+ Added subsection on how to check the results
+ Added section on measuring execution time

+ Also added links to youtube movies showing the solution evolving in time (as a motivation)

pletzer committed:
extended the intro

mand35 reviewed (commented):
I like your examples with the short videos. 
I spotted a typo on line 57: instead of "myAccount>" it should be "myAccount", the same on line 63 and 67.
And actually I would introduce how to start multi tasks/ threads job later, when using them.

Please add an example for the increased domain size and one with the checksum. People love copy and paste and actually commands are easily spotted while searching in documentation.
And Please also mention the default problem size, thus the user have an idea of what numbers they may change.

pletzer committed:
adding Mandes's suggestions

pletzer commented:
Thanks @mand35 

I fixed the typo. I moved the part describing how to run using multiple threads and MPI to the relevant sections and I added an example how to run the code using higher resolution.

pletzer committed:
minor improvements
